### PR TITLE
replace &uuml with ü

### PR DESCRIPTION
**Pull request by JustinMShea**

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -3,10 +3,11 @@ title: "About R/Finance"
 subtitle: "Applied Finance with R"
 ---
 
+
 R/Finance was founded in the fall of 2008, by a small group of open source 
 package authors, finance industry practitioners, and finance professors. 
 Inspired by popular workshops and summer schools about "R in Finance" held by 
-Diethelm W&uuml;rts at Meielisalp in the Swiss mountains, the inaugural R/Finance 
+Diethelm Würts at Meielisalp in the Swiss mountains, the inaugural R/Finance
```
Comment (eddelbuettel) [inline content/about.md]: Umlaut u is correct. Wuertz is still wrong as it ends in tz.

File: content/about.md [hunk 2]
```diff
@@ -65,7 +66,7 @@ Keynotes of R/Finance past include:
 [Dr. David Kane](https://www.davidkane.info/), 
 [Dr. Roger Koenker](http://www.econ.uiuc.edu/~roger/), 
 [Dr. David Ruppert](https://people.orie.cornell.edu/davidr/), 
-[Dr. Diethelm W&uuml;tz](https://comp.phys.ethz.ch/news-and-events/nc/2016/08/in-memoriam-diethelm-wuertz.html), 
+[Dr. Diethelm Würtz](https://comp.phys.ethz.ch/news-and-events/nc/2016/08/in-memoriam-diethelm-wuertz.html),
```
Comment (eddelbuettel) [inline content/about.md]: Correct here.

**Review by eddelbuettel — CHANGES_REQUESTED** [review]

File: content/about.md (hunk 1 above)
Comment (JustinMShea) [reply content/about.md]: Thanks! Fixed now

**Review by eddelbuettel — COMMENTED** [review]
Great, thanks!

**Review by eddelbuettel — APPROVED** [review]

**Pull request merged**